### PR TITLE
Adding a keypair generator

### PR DESCRIPTION
**Pull request by ineiti**
I hope this is correct :)

### CODE REVIEW

**Comment by PascalinDe** [comment]
LGTM

File: cli/config/config.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import (
+	"fmt"
+	"go.dedis.ch/kyber/v3/group/edwards25519"
+	"go.dedis.ch/kyber/v3/util/key"
+)
+
+func main() {
+	pair := key.NewKeyPair(&edwards25519.SuiteEd25519{})
+
+	fmt.Printf("PUBLIC_KEY=%s\nPRIVATE_KEY=%s\n", pair.Public, pair.Private)
```
Comment (pierluca) [inline cli/config/config.go:12]: Minor readability pet peeve: I'd consider doing two `Println` 

But yes, this looks about right 👍

**Review by pierluca — APPROVED** [review]

File: cli/config/config.go (hunk 1 above)
Comment (ineiti) [reply cli/config/config.go:12]: I'll close it anyway and move it to the webapp itself.

**Comment by ineiti** [comment]
Closed in favor of #353

**Pull request closed without merging**